.github/workflows/build.yml

```diff
@@ -1,9 +1,15 @@
 name: Build
 on: [pull_request]
+
 jobs:
+  files_changed:
+    uses: 'botpress/messaging/.github/workflows/files-changed.yml@master'
```

Comment:
Using `@master` here will make the checks fail, but will pass once this PR is merged since it adds this workflow